packages/report-server/src/__tests__/reportBuilder/transform/parser/parser.test.ts

```diff
@@ -121,31 +121,6 @@ describe('parser', () => {
   });
 
   describe('in transforms', () => {
-    it('mergeRows supports parser lookups on where', () => {
```

Comment:
Do we need a new test for mergeStrategy = exclude?

Reply:
Ah good call! I'll re-add them, thanks 👍

Reply:
Ah wait sorry, we do actually have the test for `exclude` removing the columns from the table

Reply:
Ah cool